site2/docs/functions-develop.md

```diff
@@ -656,6 +656,182 @@ $ bin/pulsar-admin functions create \
 
 All logs produced by `LoggingFunction` above can be accessed via the `persistent://public/default/logging-function-logs` topic.
 
+#### Customizing Function Log Level
```

Comment:
```suggestion
#### Customize Function log level
```

In Pulsar docs, normally we use the base form of a verb (for easier search) and sentence case for headings.